res/lemonstand.json

```diff
@@ -1,9 +1,9 @@
 {
     "money":0.0,
     "objective": [
-        {"Money":10000.0},
-        {"Product":[0,500,null]},
-        {"Product":[0,200,"Sold"]}
+        {"Money":1000000.0},
+        {"Product":[0,50000,null]},
+        {"Product":[0,20000,"Sold"]}
```

Comment:
This file was generated by parsing the yaml file and serializing the result into a Json, but the change is okay, don't worry.